### DISCUSSION TITLE
sqlite3.databaseerror: database disk image is malformed chia

### DISCUSSION THREAD

**Discussion by GerhardMcG** [Chia-Network/chia-blockchain#6741] [Support]
Good Day,

I am getting a sqlite3.databaseerror: database disk image is malformed in chia debug log. This is preventing me from syncing, plotting and farming. I would appreciate any support so much to get this issue resolved as soon as possible.

Regards

**Comment by automation13** [comment]
I have same issue, how to fix it?

**Comment by BrandtH22** [comment]
@GerhardMcG , the newest version(s) of chia has fixes in place to resolve this issue. Please download it from here: https://www.chia.net/downloads/

Since this issue has been resolved with the latest version(s) of chia we will be closing this ticket but if we have closed this ticket in error do not hesitate to reach out to us again with any followup questions, comments, or if the issue persists after an update.

```
Please note that upgrading major versions of chia generally requires the config.yaml to be recreated using these steps:
1. Stop chia
2. Rename the config or move it to a different directory
3. In CLI run chia init
4. Copy any custom settings from the old config to the new (plot drive directories, pool info, trusted node info, etc) - be careful here as yaml is very picky about formatting including those leading spaces
5. Start Chia
```

Note - you will also want to remove the current databases at `~\.chia\mainnet\db\` and `~\.chia\mainnet\wallet\db\` then either resync from scratch or use the official torrent:

```
1. Download the torrent file from https://www.chia.net/downloads/#database-checkpoint
2. Use a torrent client to download the full db
3. Unpack/reassemble the torrent file that was downloaded (on windows I used 7zip, Mac and linux have built in tools that work for this)
4. Move the db to the correct folder (~\.chia\mainnet\db\) and update the name to "blockchain_v2_mainnet.sqlite" (removing the date information in the name)
5. Verify the config file (~\.chia\mainnet\config\config.yaml) has the correct value under the full_node section for `database_path: db/blockchain_v2_CHALLENGE.sqlite` (should only need to change the v1 to v2)
6. Launch chia and wait for a bit (the height to hash and peers files need to be built so this can take upwards of 30 minutes)
```

The best place to reach our support team is on Discord (https://discord.gg/chia) or by reopening this ticket.
